README.md

```diff
@@ -63,7 +63,9 @@ from mqt.bench import get_benchmark
 qc_uncompiled = get_benchmark(benchmark_name="ghz", level="alg", circuit_size=5)
 
 # compile it using the MQT Predictor
-qc_compiled, compilation_information, quantum_device = qcompile(qc_uncompiled)
+qc_compiled, compilation_information, quantum_device = qcompile(
+    qc_uncompiled, devices="all", figure_of_merit="expected_fidelity"
+)
```

Comment:
This does not seem ergonomic to me.
The `qcompile` only loads the models available to it. This automatically dictates which devices are available.
One could argue that you could be training a model for 20 QPUs, but then only want to select between 3, but that is not really how the ML classifier works as it is trained on the full set of available devices (and RL models).
So I would argue that this function should not take a `devices` parameter.
I guess the other parameter is fine as there could be multiple concurrent models for various FoMs.

If you believe it is still valuable to keep the `devices` parameter around, then I'd propose to make it `str | None` and default to `None` instead of having to specify `"all"`.

Reply:
Agreed as just discussed in person.